code/game/objects/items/control_wand.dm

```diff
@@ -64,10 +68,13 @@
 					D.lock()
 			if(WAND_EMERGENCY)
 				if(D.emergency)
-					D.emergency = 0
+					D.emergency = FALSE
 				else
-					D.emergency = 1
+					D.emergency = TRUE
 				D.update_icon()
+			if(WAND_SPEED)
+				D.normalspeed = !D.normalspeed
+				to_chat(user, "Now in [D.normalspeed ? "normal" : "fast"] mode.")
```

Comment:
Needs spans here. notice will do as class.
Text also needs some more flavor. "Now in X mode" is a bit flavorless if you know what I mean